packages/node-experimental/src/sdk/initOtel.ts

```diff
@@ -27,6 +30,18 @@ export function initOtel(): () => void {
     diag.setLogger(otelLogger, DiagLogLevel.DEBUG);
   }
 
+  if (client) {
```

Comment:
Just wondering: If `client` is undefined, should we even do anything in this function?

Reply:
good question 😅 I would say right now that is more of a theoretical question, as that is just called by `init()` right after we called `initNode()`, so there should _always_ be a client. But once we eventually split this up into more easily consumable parts, we'll need to handle these cases better.

Reply:
Makes sense!